src/features/world/scenes/CornScene.ts

```diff
@@ -219,6 +208,25 @@ export class CornScene extends BaseScene {
     this.canHandlePortalHit = true;
   }
 
+  setUpSound() {
+    this.portalTravelSound = this.sound.add("portal_travel");
+
+    // Shut down the sound when the scene changes
+    // this.events.once("shutdown", () => {
+    //   this.portalTravelSound?.play({ volume: 0.5 });
+    //   if (!this.portalTravelSound || this.portalTravelSound.isPaused) {
+    //     this.sound.getAllPlaying().forEach((sound) => {
+    //       sound.destroy();
+    //     });
+    //   }
+    // });
+
+    if (!this.sound.get("nature_3")) {
+      const nature = this.sound.add("nature_3");
+      nature.play({ loop: true, volume: 0.05 });
+    }
+  }
```

Comment:
This track has been removed from the game - entering the maze currently crashes on this line.